templates/partials/page_list_item.html

```diff
@@ -20,16 +20,15 @@ <h2 class="!my-0 pb-1 font-bold !leading-none">{{ page.title }}</h2>
         {% endif %}
     </section>
     {% else %}
-    <div class="not-prose my-1 truncate"></div>
+    <div class=""></div>
     {% if page.description %}
     <p>{{ page.description }}</p>
-    {% elif page.summary %}<!---->
-    {{ page.summary | safe }}<!---->
+    {% elif page.summary %}
+    {{ page.summary | safe }}
     {% endif %}
     </div>
     {% endif %}
 
-    {% include "partials/page_info.html" %}
 
-    <a class="absolute inset-0 text-[0]" href="{{ page.permalink }}">{{ page.title }}</a>
+    <a class="cover-click absolute inset-0 text-[0]" href="{{ page.permalink }}">{{ page.title }}</a>
```

Comment:
Classes absolute inset-0 text-[0] seem to be from the old tailwind that it's not present anymore. Are they needed?

```suggestion
    <a class="cover-click" href="{{ page.permalink }}">{{ page.title }}</a>
```

Reply:
All Tailwind classes have been removed